trade-finance-manager-ui/templates/case/tasks/tasks.njk

```diff
@@ -7,6 +7,8 @@
 
 {% extends "case/case.njk" %}
 
+{% block pageTitle -%}Deal {{ deal.details.ukefDealId }} - Tasks{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.